lib/std/c/darwin.zig

```diff
@@ -124,40 +124,42 @@ pub const THREAD_NULL: thread_t = 0;
 pub const MACH_PORT_NULL: mach_port_t = 0;
 pub const MACH_MSG_TIMEOUT_NONE: mach_msg_timeout_t = 0;
 
-pub const MACH_MSG_OPTION_NONE = 0x00000000;
-
-pub const MACH_SEND_MSG = 0x00000001;
-pub const MACH_RCV_MSG = 0x00000002;
-
-pub const MACH_RCV_LARGE = 0x00000004;
-pub const MACH_RCV_LARGE_IDENTITY = 0x00000008;
-
-pub const MACH_SEND_TIMEOUT = 0x00000010;
-pub const MACH_SEND_OVERRIDE = 0x00000020;
-pub const MACH_SEND_INTERRUPT = 0x00000040;
-pub const MACH_SEND_NOTIFY = 0x00000080;
-pub const MACH_SEND_ALWAYS = 0x00010000;
-pub const MACH_SEND_FILTER_NONFATAL = 0x00010000;
-pub const MACH_SEND_TRAILER = 0x00020000;
-pub const MACH_SEND_NOIMPORTANCE = 0x00040000;
-pub const MACH_SEND_NODENAP = MACH_SEND_NOIMPORTANCE;
-pub const MACH_SEND_IMPORTANCE = 0x00080000;
-pub const MACH_SEND_SYNC_OVERRIDE = 0x00100000;
-pub const MACH_SEND_PROPAGATE_QOS = 0x00200000;
-pub const MACH_SEND_SYNC_USE_THRPRI = MACH_SEND_PROPAGATE_QOS;
-pub const MACH_SEND_KERNEL = 0x00400000;
-pub const MACH_SEND_SYNC_BOOTSTRAP_CHECKIN = 0x00800000;
-
-pub const MACH_RCV_TIMEOUT = 0x00000100;
-pub const MACH_RCV_NOTIFY = 0x00000000;
-pub const MACH_RCV_INTERRUPT = 0x00000400;
-pub const MACH_RCV_VOUCHER = 0x00000800;
-pub const MACH_RCV_OVERWRITE = 0x00000000;
-pub const MACH_RCV_GUARDED_DESC = 0x00001000;
-pub const MACH_RCV_SYNC_WAIT = 0x00004000;
-pub const MACH_RCV_SYNC_PEEK = 0x00008000;
-
-pub const MACH_MSG_STRICT_REPLY = 0x00000200;
+pub const MACH_MSG_OPTION = struct {
```

Comment:
if these definitions really all go together, could you please put them into a packed struct? the name is unfortunately not obvious based on Apple's extremely sloppy C code, so maybe the name can be determined by looking at the function(s) that these flags are passed to. Which is what? Might be nice to leave a little comment trail for the next person here.

Reply:
The reason I didn't go for a packed struct are these two definitions, which have the same value but different semantic meanings:

```c
#define MACH_SEND_ALWAYS        0x00010000      /* ignore qlimits - kernel only */
#define MACH_SEND_FILTER_NONFATAL        0x00010000      /* rejection by message filter should return failure - user only */
```

OTOH `MACH_SEND_ALWAYS` is "kernel only" so it might be ok to remove it.

They have the same value but different meanings.

Reply:
In this case, the function is `mach_msg`. Should I change the signature to accept the packed struct ?